usr/plugins/devel/caddy/src/opnsense/scripts/Pischem/Caddy/setup.sh

```diff
@@ -25,7 +26,7 @@ chown -R root:wheel "${CADDY_DIR}"
 chmod -R 750 "${CADDY_DIR}"
 
 # Create Caddy log directory
-mkdir -p "${CADDY_LOG_DIR}"
+mkdir -p "${CADDY_ACCESS_LOG_DIR}"
```

Comment:
Shouldn't this be created additionally to the "${CADDY_LOG_DIR}", because it's inside of it? I don't know if directories are created recursively.

mkdir -p "${CADDY_LOG_DIR}"
mkdir -p "${CADDY_ACCESS_LOG_DIR}"